CryptoTechReminderSystem.Test/Gateway/HarvestGatewayTests.cs

```diff
@@ -16,6 +16,10 @@ public class HarvestGatewayTests
         private const string HarvestAccountId = "123456";
         private const string UserAgent = "The Wolves";
 
+        private static readonly string DeveloperRoles =
+            "Software Engineer, Senior Software Engineer, Senior Engineer, Lead Engineer, "+
```

Comment:
```suggestion
             "Software Engineer, Senior Software Engineer, Senior Engineer, Lead Engineer, " +
            "Delivery Manager, SRE, Consultant, Delivery Principal";
```

Reply:
this suggestion should apply for both of line 20 and 21

Reply:
Done https://github.com/chubberlisk/crypto_tech/pull/31/commits/cdbe6123d78a9649118091f7a325a7f223038919.